pages/landing/enterprise-eks/index.html

```diff
@@ -7,7 +7,7 @@
   - name: Why Gruntwork
     url: /eks-for-enterprise/#why-gruntwork
 
-title: Amazon EKS for the enterprise. In about a day.
+title: Amazon EKS for the enterprise. Defined as code. In about a day.
```

Comment:
Perhaps "Managed as code" is a better fit here?

Reply:
Makes sense! Done in 2ebb6ed1fd17fc8a6d348f69904ce94b96367174